### ISSUE TITLE
Unable to change some settings

### ISSUE THREAD

**Issue by BabakAmini** [amcharts/amcharts5#349]
My problem started when I tried to fork [this code](https://codepen.io/team/amcharts/pen/xxLBymy?editors=0010) and changed it to display my data. Here is my forked code:

https://codepen.io/BabakA/pen/dyJNPoZ

As you can see, my data intervals are in seconds and, for unknown reasons and unlike the base code, AmCharts changes X-axis grids so that there is no space at the end of the series to display the end bullet and its label. The only way to display them is zoom or pan (I will disable panning and zooming out beyond the full chart in my final code).

Regardless of the reason for this, I tried to find a solution to add extra spaces to the end of the series just like the base code. As you mentioned in [this document](https://www.amcharts.com/docs/v5/reference/axisrendererx/), there are some settings like `startLocation`, `endLocation`, `location`, and `cellEndLocation `that seem to be effective in the location of the last point in the series. I used them when I was creating the axis but nothing happened. Trying to change those settings (by using `xAxis.get("renderer").setAll)` also didn't help. Finally, I tried to test other non-location settings such as `fill`, `width`, `height`, and `stroke `to see if they could make a difference, but still to no avail.

**So, what can I do to solve this problem?**

Also, I ask you to please prepare a clearer document. Some settings, like `fill` seems to have no effect at the creation time. Also, after creating the object there is no way to change them. However, we can change fill setting of the axis' children like these:

`xAxis.get("renderer").labels.template.set("fill", am5.color(0xff0000))`
`xAxis.get("renderer").grids.template.set("fill", am5.color(0xff0000))`

I know that the fill setting came from the Graphics base class and I guess it has no meaning when rendering X-axis. So, isn't it better to change the base classes?

**Comment by martynasma** [comment]
How about just adding some extra scale to the X axis using `extraMax` setting?

```JavaScript
var xAxis = chart.xAxes.push(
  am5xy.DateAxis.new(root, {
    extraMax: 0.05,
    // ....
```

More info:
https://www.amcharts.com/docs/v5/charts/xy-chart/axes/date-axis/#relative-scope-extension

**Comment by BabakAmini** [comment]
@martynasma As always, you provide the best guides, I appreciate it.

I tried your solution, It works for an initial view of the chart, but when I zoom in, an extra time is added to the end of the axis. I also tried to set `max` setting to **12:32** or something like that but didn't help. Any suggestions for this?

![image](https://user-images.githubusercontent.com/85762931/160134780-ada5535f-e9c2-40a5-943b-1ecc4f9297e1.png)
